server/src/interfaces/search.interface.ts

```diff
@@ -57,12 +58,12 @@ export interface SearchStatusOptions {
   isEncoded?: boolean;
   isFavorite?: boolean;
   isMotion?: boolean;
-  isOffline?: boolean;
   isVisible?: boolean;
   isNotInAlbum?: boolean;
   type?: AssetType;
   withArchived?: boolean;
   withDeleted?: boolean;
+  trashReason?: AssetTrashReason | null;
```

Comment:
Why is it nullable here? Where's the difference between undefined and null for `trashReason` here?

Reply:
Fixed!